packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -85,13 +88,14 @@ const BLOCK_EDITOR_SETTINGS = [
 /**
  * React hook used to compute the block editor settings to use for the post editor.
  *
- * @param {Object} settings EditorProvider settings prop.
- * @param {string} postType Editor root level post type.
- * @param {string} postId   Editor root level post ID.
+ * @param {Object} settings        EditorProvider settings prop.
+ * @param {string} postType        Editor root level post type.
+ * @param {string} postId          Editor root level post ID.
+ * @param {string} contextPostType The post type of the edited post.
  *
  * @return {Object} Block Editor Settings.
  */
-function useBlockEditorSettings( settings, postType, postId ) {
+function useBlockEditorSettings( settings, postType, postId, contextPostType ) {
```

Comment:
I preferred the way @draganescu set the client id in https://github.com/WordPress/gutenberg/pull/60609, but there's some feedback in the other PR about changing this function signature, so I'd like to make the new signature part of a private API so we can play around with it until we find something that we like.

Reply:
Actually this entire function is already part of the private APIs, so it should be fine.